### ISSUE TITLE
v4: PHP Warning:  array_merge(): Expected parameter 1 to be an array, null given in /path/to/vendor/phptek/sentry/src/Adaptor/SentryAdaptor.php on line 157

### ISSUE THREAD

**Issue by elliot-sawyer** [phptek/silverstripe-sentry#71]
```
PHP Warning:  array_merge(): Expected parameter 1 to be an array, null given in /path/to/vendor/phptek/sentry/src/Adaptor/SentryAdaptor.php on line 157
PHP Fatal error:  Uncaught TypeError: Return value of PhpTek\Sentry\Adaptor\SentryAdaptor::get_opts() must be of the type array, null returned in /path/to/vendor/phptek/sentry/src/Adaptor/SentryAdaptor.php:171
```

I have SENTRY_DSN defined in my .env and also tried inserting via Injector, but getting the above error on v4.   Rolling back to 3.0.10 resolves the issue. 

Calling dump/die of `$optsConfig` shows that it returns `null`, presumably because `private static opts`; is not defined. This will cause `array_merge` to fail

**Comment by elliot-sawyer** [comment]
PR here https://github.com/phptek/silverstripe-sentry/pull/72

**Comment by phptek** [comment]
Closing. Merged in #73
